pandas/core/frame.py

```diff
@@ -4486,6 +4489,12 @@ def _series_round(s, decimals):
                 return s.round(decimals)
             return s
 
+        validate_args(args, min_length=0, max_length=1)
+        out = args[0] if args else None
+
+        if out is not None:  # Series.round has justification
```

Comment:
I would like this check to occur in ``validate_args`` actually, or make another validation function.

we are going to have to sprinkle these to multiple functions, so want to make it easy (actually maybe the best here is to have a ``core/base.py`` method something like:

``_validate_compat_args`` or somesuch that auto validates ``out``.